oxidation/libparsec/crates/types/src/addr.rs

```diff
@@ -319,10 +332,13 @@ impl BackendAddr {
         self.base.to_http_url(path)
     }
 
-    fn _from_url(parsed: &Url, pairs: &url::form_urlencoded::Parse) -> Result<Self, AddrError> {
-        let base = BaseBackendAddr::from_url(parsed, pairs)?;
+    fn _from_url(
+        parsed: &ParsecUrlAsHTTPScheme,
+        _pairs: &url::form_urlencoded::Parse,
```

Comment:
If you don't need it, remove it ;)

Reply:
it is needed by the other `_from_url` (given they all got called by the same common macro)

Reply:
Seems like a trait would be nice ;)

Reply:
could be, but this code is "fine enough" given we won't touch it much now that it is stable